File: Day25/frontend/src/domain/entities/Submission.ts
// domain/entities/Submission.ts
// Pure business types. No imports from React, Redux, or Axios —
// this file would survive a full framework rewrite untouched.

export interface EducationEntry {
  institution: string;
  degree: string;
  year: number;
}

export interface Submission {
  id: string;
  fullName: string;
  email: string;
  phone: string;
  education: EducationEntry[];
  fileUrl: string;
  createdAt: string;
}

File: Day28/Form(backend)/Entities/Submission.cs
namespace Form.Entities;

public class Submission
{
    public Guid Id { get; set; } = Guid.NewGuid();
    public string FullName { get; set; } = string.Empty;
    public string Email { get; set; } = string.Empty;
    public string Phone { get; set; } = string.Empty;
    public string FileUrl { get; set; } = string.Empty;
    public DateTime CreatedAt { get; set; } = DateTime.UtcNow;

    public ICollection<EducationEntry> Education { get; set; } = new List<EducationEntry>();
}
